scripts/deploy-frontend.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/sh
+
+# source deploy-util.sh
+
+# The deployment strategy you wish to employ ( rolling update or setting up a new environment)
+DEPLOY_STRATEGY=${1}
+
+# The environment in which you want to execute these commands
+DEPLOY_ENV=${2}
+
+#The application name  defined via the manifest yml for the frontend
+CGHOSTNAME_FRONTEND=${3}
+CGHOSTNAME_BACKEND=${4}
+
+
+update_frontend()
+{
+
+
+    echo DEPLOY_STRATEGY: $DEPLOY_STRATEGY
+    echo DEPLOY_ENV: $DEPLOY_ENV
+    echo FRONTEND_HOST: $CGHOSTNAME_FRONTEND
+    echo BACKEND_HOST: $CGHOSTNAME_BACKEND
+    cd tdrs-frontend
+
+    echo "REACT_APP_BACKEND_URL=https://$CGHOSTNAME_BACKEND.app.cloud.gov/v1" >> .env.production
+    echo "REACT_APP_BACKEND_HOST=https://$CGHOSTNAME_BACKEND.app.cloud.gov" >> .env.production
+    npm run build
+    unlink .env.production
+    mkdir deployment
+
```

Comment:
We decided to create a temporary deployment folder to avoid sending all of the project files.
The dockerfile originally only included the production build, leaving out the source code and node_modules, and ran nginx.

Using the nginx build pack, all files included in the directory you run the command from get included, and we were going over the disk quota. So now we build it, copy the build, copy the nginx configs, copy the mainifest, run the deployment command, then delete the deployment folder.